core/types.go

```diff
@@ -32,13 +32,14 @@ type DutyType int
 const (
 	// DutyType enums MUST not change, it will break backwards compatibility.
 
-	DutyUnknown  DutyType = 0
-	DutyProposer DutyType = 1
-	DutyAttester DutyType = 2
-	DutyRandao   DutyType = 3
+	DutyUnknown       DutyType = 0
+	DutyProposer      DutyType = 1
+	DutyAttester      DutyType = 2
+	DutyRandao        DutyType = 3
+	DutyVoluntaryExit DutyType = 4
```

Comment:
I would propose we rename this to `DutyExit`, since there is only this type of exit duty, there are no non-voluntary or any other type of exit duty. This also aligns with the other duties above, which is just `DutyProposer` and `DutyRandao` instead of more verbose redundant versions.

Reply:
I don't fell comfortable with this, there will be other Exits in the code, could we ask for feedback to @OisinKyne @dB2510 @xenowits ?